mlflow/azureml/__init__.py

```diff
@@ -207,7 +210,180 @@ def build_image(model_uri, workspace, image_name=None, model_name=None,
         return image, registered_model
 
 
-def _build_tags(model_uri, model_python_version=None, user_tags=None):
+@experimental
+def deploy(model_uri, workspace, deployment_config=None, service_name=None, model_name=None,
+           mlflow_home=None, synchronous=True):
+    """
+    Register an MLflow model with Azure ML and deploy a websevice to Azure Container Instances (ACI)
+    or Azure Kubernetes Service (AKS).
+
+    The deployed service will contain a webserver that processes model queries.
+    For information about the input data formats accepted by this webserver, see the
+    :ref:`MLflow deployment tools documentation <azureml_deployment>`.
+
+    :param model_uri: The location, in URI format, of the MLflow model used to build the Azure
+                      ML deployment image. For example:
+
+                      - ``/Users/me/path/to/local/model``
+                      - ``relative/path/to/local/model``
+                      - ``s3://my_bucket/path/to/model``
+                      - ``runs:/<mlflow_run_id>/run-relative/path/to/model``
+                      - ``models:/<model_name>/<model_version>``
+                      - ``models:/<model_name>/<stage>``
+
+                      For more information about supported URI schemes, see
+                      `Referencing Artifacts <https://www.mlflow.org/docs/latest/concepts.html#
+                      artifact-locations>`_.
+
+    :param service_name: The name to assign the Azure Machine learning webservice that will be
+                       created. If unspecified, a unique name will be generated.
+    :param model_name: The name to assign the Azure Model will be created. If unspecified,
+                       a unique model name will be generated.
+    :param workspace: The AzureML workspace in which to deploy the service. This is a
+                      `azureml.core.Workspace` object.
+    :param deployment_config: The configuration for the Azure web service. This configuration
+                              allows you to specify the resources the webservice will use and
+                              the compute cluster it will be deployed in. If unspecified, the web
+                              service will be deployed into a Azure Container Instance. This is a
+                              `azureml.core.DeploymentConfig` object. For more information, see
+                              `<https://docs.microsoft.com/python/api/azureml-core/
+                              azureml.core.webservice.aks.aksservicedeploymentconfiguration>` and
+                              `<https://docs.microsoft.com/en-us/python/api/azureml-core/azureml.core.webservice.aci.aciservicedeploymentconfiguration>
```

Comment:
These documentation links don't seem to be rendering properly:

<img width="617" alt="Screen Shot 2020-04-10 at 5 55 24 PM" src="https://user-images.githubusercontent.com/39497902/79031692-4ea38f80-7b55-11ea-9e96-ab066dbd6996.png">

Here's a fix:
```suggestion
                              `<https://docs.microsoft.com/python/api/azureml-core/
                              azureml.core.webservice.aks.aksservicedeploymentconfiguration>`_ and
                              `<https://docs.microsoft.com/en-us/python/api/azureml-core/azureml
                              .core.webservice.aci.aciservicedeploymentconfiguration>`_
```